### PR TITLE
Wazuh-kibana-app 5.0 agent process unification compliant

### LINKED ISSUE #3123: Wazuh-kibana-app 5.0 agent process unification compliant.
## Description
This issue aims to modify the way in which verifications are made or Wazuh core processes are used in the internal workings of this repository, given the epic: https://github.com/wazuh/wazuh/issues/7981

For this, the following repository occurrences are listed so that they can be analyzed, modified and/or adapted based on the requirement.

Current: execd, logcollector and syscheckd exist in the Wazuh ecosystem.
Expected: unify the old process behavior under the guardianship of wmodulesd. 
Versions in which it should be applied: >= 5.0 

**Occurrences:**
https://github.com/wazuh/wazuh-kibana-app/search?q=execd
https://github.com/wazuh/wazuh-kibana-app/search?q=syscheckd
https://github.com/wazuh/wazuh-kibana-app/blob/34a7422668f5880ee75b0ba9ec0f5cbd91c1fe7e/server/lib/api-request-list.json
https://github.com/wazuh/wazuh-kibana-app/blob/34a7422668f5880ee75b0ba9ec0f5cbd91c1fe7e/common/api-info/endpoints.json

### PR DESCRIPTION
**Pull request by CPAlejandro**
Hi guys,
We have changed all execd, logcollector and syscheckd to wsystemd.
Closes #3123

### CODE REVIEW

**Review by mpRegalado — APPROVED** [review]
CR: LGTM! :heavy_check_mark: 
Testing: LGTM! :heavy_check_mark: 
Launched a 5.0 environment and browsed through the app with no visible errors.

**Review by pablomarga — APPROVED** [review]
CR: :heavy_check_mark: 
Testing: :heavy_check_mark:

File: public/components/agents/stats/agent-stats.tsx [hunk 1]
```diff
@@ -107,15 +107,15 @@ export const MainAgentStats = compose(
...
   const [loading, setLoading] = useState();
-  const [dataStatLogcollector, setDataStatLogcollector] = useState({});
+  const [dataStatwmodulesd, setDataStatwmodulesd] = useState({});
   const [dataStatAgent, setDataStatAgent] = useState();
   useEffect(() => {
     (async function(){
       setLoading(true);
       try{
-        const responseDataStatLogcollector = await WzRequest.apiReq('GET', `/agents/${agent.id}/stats/logcollector`, {});
+        const responseDataStatwmodulesd = await WzRequest.apiReq('GET', `/agents/${agent.id}/stats/wmodulesd`, {});
```
Comment (frankeros) [inline public/components/agents/stats/agent-stats.tsx:116]: the stats endpoints don't change, please rollback the changes for this module

**Review by frankeros — CHANGES_REQUESTED** [review]

File: public/components/agents/stats/agent-stats.tsx (hunk 1 above)
Comment (CPAlejandro) [reply public/components/agents/stats/agent-stats.tsx]: I changed it, thank you!

**Review by frankeros — APPROVED** [review]

**Pull request merged**